src/components/panel/demoBasicUsage/style.css

```diff
@@ -0,0 +1,25 @@
+.md-panel-demo {
+  min-height: 500px;
+}
+
+.dialog-example {
+  border-radius: 4px;
+  box-shadow: 0 7px 8px -4px rgba(0, 0, 0, 0.2),
+      0 13px 19px 2px rgba(0, 0, 0, 0.14),
+      0 5px 24px 4px rgba(0, 0, 0, 0.12);
+  width: 500px;
+}
+
+.dialog-content {
+  padding: 0 15px;
+  width: 100%;
+}
+
+.dialog-content img {
+  height: 300px;
+  margin: auto;
+}
+
+.dialog-button {
+  width: 100%;
+}
```

Comment:
Just thought of this- could you prefix these css classes with `demo-`, so that it's clear what is from the demo and what is part of the underlying component?